src/corehost/cli/comhost/clsidmap.cpp

```diff
@@ -45,29 +45,21 @@ namespace
 
     clsid_map parse_stream(_Inout_ pal::istream_t &json_map_raw)
     {
-        skip_utf8_bom(&json_map_raw);
+        json_parser_t json;
 
-        // Parse JSON
-        json::value json_map;
-        try
-        {
-            json_map = json::value::parse(json_map_raw);
-        }
-        catch (const json::json_exception&)
+        if (!json.parse_stream(json_map_raw, "<embedded .clsidmap>"))
```

Comment:
I think you are going to need `_X("<embedded .clsidmap>")`

Reply:
Yes, indeed. Fixed.